modules/kubeflow-mlflow/main.tf

```diff
@@ -58,8 +59,9 @@ module "kubeflow" {
 }
 
 module "mlflow" {
-  source = "git::https://github.com/canonical/charmed-mlflow-solutions//modules/mlflow?ref=track/2.15"
+  source = "git::https://github.com/canonical/charmed-mlflow-solutions//modules/mlflow?ref=track/2.22"
```

Comment:
I agree that this change makes sense, but it feels a bit out of scope for this PR. Would it make more sense to address it in a separate one for better visibility and traceability? I think a change like updating the mlflow component version for the solution should be done in its own PR.

Reply:
Yes, makes sense. Raise https://github.com/canonical/charmed-kubeflow-solutions/pull/61